index.js

```diff
@@ -37,12 +37,14 @@ function HttpServer({ listeningPort, rootFolder, sslConfig }, callback) {
 	const server = new Server(sslConfig);
 	server.rootFolder = rootFolder;
 
-	checkPortInUse(port, sslConfig, (err, status) => {
+	checkPortInUse(port, sslConfig, (_, status) => {
 		if (status === true) {
-			throw Error(`Port ${port} is used by another server.`);
+			const err = new Error(`Port ${port} is used by another server.`);
+			err.code = 'EADDRINUSE';
```

Comment:
Set the exception code before throwing it in order for the parent process to detect this issue and restart the server